### DISCUSSION TITLE
so-soc has gone AWOL

### DISCUSSION THREAD

**Discussion by Silvrak** [Security-Onion-Solutions/securityonion#7953] [Unsupported Versions] [answered]
env = SO 2.3.120 - Ubuntu Server 18.04.6 - In production over 6 months with 0 issues as a standalone server.

Performed soup, when the server came back online after a reboot I could not connect to web console, starting receiving "401 Authorization Required."
Running so-status reveals all containers with status ok, except for so-soc, which has status of missing.
Logs in /opt/so/log/soc/ pointed me to: error="unexpected end of JSON input" file=jobs/soserv/1044.json
" "1045.json
" "1046.json
the .json files are completely devoid of any code...no content at all.
running so-docker-refresh runs successfully but after rebooting the server and testing, the issue does not get fixed.
At this point, I am considering a rebuild but would rather not have too. Any advice?

**Comment by jertel** [comment, marked as answer]
Remove the empty `jobs/.../*.json` files and then run `so-soc-restart`. I would guess that an improper shutdown of your server could have contributed to those corrupted files.

**Reply by Silvrak** [reply]
Removed the files per suggestion, but no joy. At this point I am going to have to rebuild as this needs to be online.  Thank you for the reply.
